src/store/actions/updateEnabledAssets.js

```diff
@@ -0,0 +1,13 @@
+import buildConfig from '../../build.config'
+import { Networks } from '../utils'
+
+export const updateEnabledAssets = async ({ state, dispatch, getters }) => {
```

Comment:
wouldn't this renable assets even if you disabled them in settings?

Reply:
removed, thanks!